packages/engine-server/src/workspace/service.ts

```diff
@@ -135,6 +135,10 @@ export class WorkspaceService {
     return path.join(this.wsRoot, "dendron");
   }
 
+  get seedService(): SeedService {
```

Comment:
@kevinslin let me know if this is an ok way to expose SeedService, or if you had something else in mind.

Reply:
yeah, looks good